source/utilities.tex

```diff
@@ -14819,39 +14846,60 @@
 string s4 = format("{:*^6}", 'x');      // value of \tcode{s4} is \tcode{"**x***"}
 string s5 = format("{:6d}", c);         // value of \tcode{s5} is \tcode{"\ \ \ 120"}
 string s6 = format("{:6}", true);       // value of \tcode{s6} is \tcode{"true\ \ "}
+string s7 = format("{:*<6.3}", "123456");   // value of s7 is "123***"
+string s8 = format("{:02}", 1234);          // value of s8 is "1234"
+string s9 = format("{:*<}", "12");          // value of s9 is "12"
+string sA = format("{:*<6}", "12345678");   // value of sA is "12345678"
 \end{codeblock}
```

Comment:
See https://github.com/cplusplus/draft/pull/6139#discussion_r1114664091. This is so that future additions are known to follow the sequence.
```suggestion
%% FIXME: string sB = ... string sB from P2572R1 ... // value of \tcode{sB} is ...
%% FIXME: string sC = ... string sC from P2572R1 ... // value of \tcode{sC} is ...
\end{codeblock}
```

Reply:
Unfortunately we can't do that (have a FIXME inside the `\codeblock`) because it would appear in the final paper, and I'm pretty sure we don't want that.